### ISSUE TITLE
Cannot download firefox-legacy-1.16.4.30

### ISSUE THREAD

**Issue by digagithub** [gorhill/uBlock-for-firefox-legacy#338]
I cannot download "firefox-legacy-1.16.4.30".

Clicking on "Assets" there is will be the turning symbol but nothin happens.

Please can anyone check whether he succeeds ... ?

Thanks!

**Comment by mapx-** [comment]
working fine on my end

https://github.com/gorhill/uBlock-for-firefox-legacy/releases/tag/firefox-legacy-1.16.4.30

clicking [uBlock0_1.16.4.30.firefox-legacy.xpi](https://github.com/gorhill/uBlock-for-firefox-legacy/releases/download/firefox-legacy-1.16.4.30/uBlock0_1.16.4.30.firefox-legacy.xpi)

**Comment by AroKol78** [comment]
install the add-on first https://martok.github.io/palefill/ solves problems with github

> Clicking on "Assets" there is will be the turning symbol but nothin happens.

description of the problem https://github.com/martok/palefill/issues/47

**Comment by Gittyperson** [comment]
Asked and answered several times already.

https://github.com/gorhill/uBlock-for-firefox-legacy/issues/335

**Comment by digagithub** [comment]
Thanks, palefill works.

But why so complicated?

**Comment by hujan86** [comment]
> Thanks, palefill works.
> 
> But why so complicated?

Github uses Google's WebComponents now, something that browsers other than Chrome, Firefox and its derivatives doesn't or only partially supports.

**Comment by HexagonWin** [comment]
I saw this some time ago from somewhere, very useful.
For Github release, on web browsers not supporting WebComponents, you can change "tag" in the url to "expanded_assets" and make it show.

For example : https://github.com/gorhill/uBlock-for-firefox-legacy/releases/tag/firefox-legacy-1.16.4.30 => https://github.com/gorhill/uBlock-for-firefox-legacy/releases/expanded_assets/firefox-legacy-1.16.4.30

**Comment by FrostKnight** [comment]
> > Thanks, palefill works.
> > But why so complicated?
> 
> Github uses Google's WebComponents now, something that browsers other than Chrome, Firefox and its derivatives doesn't or only partially supports.

Yeah, because chrome is all that matters and totally isn't the most bloated awful evil web browser out there.  Smh...

That being said,  changing from releases to tags...

I figured this out a few years ago more or less.  I thought people knew about this already.

Pity that the web standards that websites seem to support are the most bloated piles of crap to hit the fan and that the ones that are reasonable, are looked at as if they were absolute abominations that no one should ever touch.

Javascript =  could it get much worse?
java =  absolute abomination
anything similar = wretched possible worse or slightly better depending on the situation.
bloated software that the web runs on in general when better alternatives exist  = huge pile of crap that should be sent into a void in general.

Irony...

The above rant is sponsored by:

Bullcrap! "If it ain't crap, big business usually won't even come near it"

This all being said, if anyone finds a ublock origin legacy fork that is being worked on still, you'll have my interest.

**Comment by Gittalax** [comment]
+1
that "improvement" (too) was totally unnecessary. Does not need fix that works already. Maybe it was not enough fancy and it is the most important these days.

**Comment by FrostKnight** [comment]
> +1 that "improvement" (too) was totally unnecessary. Does not need fix that works already. Maybe it was not enough fancy and it is the most important these days.

Funny thing, the forum software "**discourse**" claims to solve a problem of helping web browsers adopt the new javascript standards. There is just an obvious issue though which is, that they are putting a bandaid on a much worse problem.

The solution is  to stop using javascript for the web and not the current way of doing things which is bloat that the creators of java would probably be proud of seeing come to life.

The answer should be screw fancy graphics and:

The web called, it says, "I want my a reasonable standard! Not some bloated piece of crap!"


This all being said, I would be all too thrilled if the graphics of the web went back to before java screwed stuff up, etc... and then if fancy graphics are needed, do it in a way that doesn't favor the corporate beasts of the world.

The more humans waste electricity and cause hardware to die due to avoidable planned obsolescence and stuff similar, the more the climate dies..

Simply put, facepalm.jpeg is more then appropriate here.

It is a friggin foregone conclusion.

;)
